target/linux/mvebu/files/arch/arm/boot/dts/armada-380-iij-sa-w2.dts

```diff
@@ -66,12 +66,14 @@
 		pinctrl-0 = <&pmx_leds_pins>;
 
 		led-0 {
+			label = "green:wlan5g";
```

Comment:
```
function = LED_FUNCTION_WLAN;
function-enumerator = <5>;
```
and similarly further

Reply:
I thought about using them once, but it might be confusing for users without the suffix 'g', so I used label instead.

Reply:
We already have mixed variants across the tree.
0/1 (matching the interface names), 2/5, 24/50 and labels with “g“.

I personally do have a light preference  for using color/function for generic names, but I do see valid arguments for all versions. If frequencies are fixed on the radios, I’d be fine with either. Otherwise I’d use 0/1/.. and just map the label to the radio name.

Reply:
@rmilecki added separate LED functions for 2.4, 5 and 6 GHz Wifi to upstream kernel, see:
https://github.com/torvalds/linux/commit/ec18a2a83b8b9f7e39c80105ea148c769c46227b